### DISCUSSION TITLE
[zeusing] - Inhibit an area/location from despawning?

### DISCUSSION THREAD

**Discussion by OverlordZorn** [Vdauphin/HeartsAndMinds#1509] [Question and answer] [answered]
I use Zeus in HAM alot to create scenarios and so on, but often, when i place opfor units I kiind of needto stay in the area as zeus to keep the nearest town initialised, otherwise i risk for the town to "store" all nearby units, including those placed by zeus.

Is there a way to keep a town initialised? for example a specific object that i can place nearby?

**Comment by Vdauphin** [comment, marked as answer]
Hello,
You could change "ANYPLAYER" by "WEST" here
https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/fnc/city/setPlayerTrigger.sqf#L30
and place a west unit nearby
Cheers

**Reply by OverlordZorn** [reply]
Amazing, Thanks!
